### DISCUSSION TITLE
Apply to GitHub Global Campus as a teacher

### DISCUSSION THREAD

**Discussion by epageperron** [community/community#54324] [GitHub Education]
### Hi everyone,

I'm trying to get authenticated as a teacher. I'm using the same email as last time when I was approved. I tried many different documents with no success. I even tried with my pay slip! My college which is part of the email name is not in the list of the "pre-approved" colleges for my uni, this might be a contributing factor. Any suggestions on what document to send would be appreciated.

**Comment by samus-aran** [comment]
Hi @epageperron  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by epageperron** [reply]
Could you add my college to the list of colleges for the University of Oxford emails? A lot of students affiliated with my college are also complaining that they are not able to authenticate. Our emails end by wolfson.ox.ac.uk . All emails .ox.ac.uk are from the University of Oxford in all cases. Here is the list of all colleges at the University of Oxford. Most are on your list of accepted emails. https://www.ox.ac.uk/admissions/undergraduate/colleges/a-z-of-colleges  and 
 https://www.ox.ac.uk/admissions/graduate/colleges/college-listing. My college, Wolfson, only accepts graduate students no undergraduates so it appears in the second list only.

**Reply by epageperron** [reply]
https://www.wolfson.ox.ac.uk/
